Client/Frontend/Browser/TabManager.swift

```diff
@@ -606,7 +604,7 @@ extension TabManager {
                 }
             }
         }
-        guard count == 0, !DebugSettingsBundleOptions.skipSessionRestore, store.hasTabsToRestoreAtStartup else {
+        guard count == 0, !AppConstants.IsRunningTest, !DebugSettingsBundleOptions.skipSessionRestore, store.hasTabsToRestoreAtStartup else {
```

Comment:
This is how the code was originally (I had moved the IsRunningTest to bypass the whole function).